.gitattributes

```diff
@@ -11,11 +11,31 @@ docs/visibility.md linguist-generated=true
 docs/web-ui.md linguist-generated=true
 docs/workers.md linguist-generated=true
 docs/workflows.md linguist-generated=true
-docs/application-development/debugging.md linguist-generated=true
-docs/application-development/features.md linguist-generated=true
-docs/application-development/foundations.md linguist-generated=true
-docs/application-development/observability.md linguist-generated=true
-docs/application-development/testing.md linguist-generated=true
+docs/dev-guide/golang/debugging.md linguist-generated=true
```

Comment:
I get the case for typescript over ts, but think that just go would be better here

Reply:
👍